docs/src/content/docs/getting-started/debugging.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Please, fill this page with the original content (https://reatom-git-structured-docs-artalar.vercel.app/getting-started/debugging/) or mix it together